crates/red_knot_python_semantic/src/semantic_index/builder.rs

```diff
@@ -284,8 +299,9 @@ impl<'db> SemanticIndexBuilder<'db> {
                         ..
                     }) => (name, &None, default),
                 };
-                // TODO create Definition for typevars
-                self.add_or_update_symbol(name.id.clone(), SymbolFlags::IS_DEFINED);
+                let symbol = self.add_symbol(name.id.clone());
+                // TODO create Definition for PEP 695 typevars
+                self.mark_symbol_bound(symbol);
```

Comment:
Totally different meaning of the same word :/ The "bound" here is the "bounds" (i.e. constraints) on the possible types of the TypeVar, which has nothing to do with whether or not a name is "bound".

I don't really want to totally rename from "bound" here, since the AST and everything uses that term. I'll add a comment to clarify. Also, when we fix the TODO here and create definitions for typevars, then this `mark_symbol_bound` call will go away (it will happen implicitly inside `add_definition` instead), which will reduce the double use of the term here.